stories/ClaimStatus.stories.tsx

```diff
@@ -10,4 +10,7 @@ export default {
 const Template: Story<ClaimStatusProps> = (args) => <ClaimStatusComponent {...args} />
 
 export const ClaimStatus = Template.bind({})
-ClaimStatus.args = {}
+ClaimStatus.args = {
+  statusDescription: 'claim-status:base-pending.description',
```

Comment:
Note: I'm wondering if it would be useful to convert this into a radio button or drop down selection in the Storybook. Thoughts?

Reply:
I think it sounds nice, if its not a huge lift!